.github/ISSUE_TEMPLATE/02-chaosscast-episode-checklist.md

```diff
@@ -0,0 +1,89 @@
+  * [ ] **Action:** Add date of shipment on [episode overview table](https://docs.google.com/document/d/1qxmC1k8SayChurIq2r91CR3r_KAThTwaNEMjA6DLGhE/edit) to confirm it was sent
+
+### Recording an episode
+
+* On day of podcast \(Organizer and Panelists\):
+  * [ ] **Action:** Review the [CHAOSScast Recording Procedures](https://docs.google.com/document/d/1G_CFd8CqwxLCahLrC-4XRzaad0OdLX43AY_--UPK2lo/edit#) before the session starts
+    * Feel good about yourself and get excited!
+* When podcast session starts \(Organizer and Panelists\):
+  * Join the Zoom room on time
+  * [ ] **Action:** Confirm that the podcast is being recorded.
+  * Be yourself
+  * Enjoy
+
+### Publishing an episode \(~30-40 min\)
+
+* Paul Bahr from Peachtree Sound will send the edited podcast and show notes \(email\) **After podcast is edited**
+  * [ ] **Action:** Get edited podcast and show notes from Paul
+  * [ ] **Action:** Review the Episode Planning Document for the guest's profile pic, bio, and social media links.
+  * [ ] **Action:** Create the Cover Art by downloading a sample [cover art from Dropbox](https://www.dropbox.com/sh/l7iv7qewxk5ud8o/AADj2oAVfg2brvd8DulJPxEla/Art?dl=0&subfolder_nav_tracking=1) and updating the episode number
+  * [ ] **Action:** Upload the shownotes to google drive and use Convert to Markdown to get markdown format
+  * [ ] **Action:** In [Fireside](https://chaosscast.fireside.fm/admin/), create missing panelists and guests
+    * Slug is `firstname-lastname` \(e.g., `georg-link`\)
+    * Status: public
+    * The other information should be in the Episode Planning Document
+  * [ ] **Action:** In [Fireside](https://chaosscast.fireside.fm/admin/), create a new episode
+    * Permalink: \# \(only the number of episode\)
+    * Visibility: Public
+    * Title: Title from the Episode Planning Document
+    * Title Format: Episode 123: My Great Title \(default\)
+    * Content: Clean \(unless we used swear words\)
+    * Summary: First paragraph from the Show Notes are a good starting point
+    * Description: Insert the show notes in Markdown format.
+      * Hint: convert show notes to Google Doc and export as Markdown
+    * Keywords: \(use sparingly to avoid keyword cannibilization\)
+    * Tags: \(empty\) -- we can use tags later for having sub-podcasts
+    * Hosts and Guests: activate everyone who was on the show. Regular panelists are all hosts, even when they are guest on an episode.
+    * Social Media Destinations: \(ignore\)
+    * Apple Podcasts Settings: \(ignore\)
+    * MP3 File: upload [MP3 file provided by CPN through Dropbox](https://www.dropbox.com/sh/l7iv7qewxk5ud8o/AAC8aNnZYbKy6K6gsdg5pfTGa/Episodes?dl=0&subfolder_nav_tracking=1)
+    * Cover Art: upload [cover art from Dropbox](https://www.dropbox.com/sh/l7iv7qewxk5ud8o/AADj2oAVfg2brvd8DulJPxEla/Art?dl=0&subfolder_nav_tracking=1) with the correct episode number
+    * Header Image: \(ignore, will use default\)
+    * Transcript: \(ignore\)
+  * PUBLISH
```

Comment:
In the publish workflow, include sending a Thank You email to the guest with information about when the episode will be released and what the link will be. That email can also include a link to a webform for requesting a physical thank-you card.